lib/meteor/meteorProcess.js

```diff
@@ -75,7 +75,7 @@ module.exports = function createMeteorProcess (node, main, env, myPrototype, opt
         }, startupTimeout);
       }
 
-      if (/Поехали!/.test(data.toString())) {
+      if (/Gagarin: Ready! Tests may engage!/.test(data.toString())) {
```

Comment:
@RobertLowe Are you aware that this was in fact an easter egg?

Reply:
I am, and I love it, but I changed it because it scared the crap out of a non-technical co-worker, they thought we were hax'd. XD

I have no opinions on it, it does seem a bit clearer in at least providing the source (gagarin) it's from.